.github/workflows/release.yaml

```
@@ -83,7 +83,7 @@ jobs:
           push: true
           build-args:
             BACKBEAT_VERSION=${{ github.event.inputs.tag }}
-          tags: ghcr.io/${{ github.repository }}:${{ github.sha }}-federation
+          tags: ghcr.io/${{ github.repository }}:${{ github.event.inputs.tag }}-federation
```

`github.event.inputs` is deprecated, use `inputs`

```suggestion
          tags: ghcr.io/${{ github.repository }}:${{ inputs.tag }}-federation
```